app/app.go

```diff
@@ -167,7 +167,7 @@ var (
 		coinswaptypes.ModuleName:       {authtypes.Minter, authtypes.Burner},
 		servicetypes.DepositAccName:    {authtypes.Burner},
 		servicetypes.RequestAccName:    nil,
-		servicetypes.TaxAccName:        {authtypes.Burner},
```

Comment:
need to migrate

Reply:
Add an issue to track it, it should be done in the upgrade process via another PR.